rabbitmq/templates/deployment.yaml.j2

```diff
@@ -1,24 +1,20 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 spec:
-  replicas: 1
+  replicas: 3
   template:
     metadata:
       labels:
         app: rabbitmq
     spec:
       nodeSelector:
         {{ deployment.control_node_label }}: enabled
-      # TODO(DTadrzak): it must be removed in future
-      securityContext:
-        runAsUser: 0
       containers:
         - name: rabbitmq
+          command:
```

Comment:
In order to make it more consistent with other services, image should set kubernetes-entrypoint as a entrypoint by default, then you don't need to pass it via command.